composer.json

```diff
@@ -6,11 +6,12 @@
 
     , "require": {
           "php": "^5.5|^7.0"
-        , "illuminate/database": "~5.2.0"
-        , "symfony/dependency-injection": "^2.6|^3.0"
+        , "illuminate/database": "^5.0.0 <5.3"
+        , "symfony/dependency-injection": "^2.8|^3.0"
     }
     , "require-dev": {
           "matthiasnoback/symfony-dependency-injection-test": "^0.7"
+        , "phpunit/phpunit": "^4.8|^5.0"
```

Comment:
This real use case is fair and it's when you have to be pragmatic, but another real usage is you using a dependency used by PHPUnit, and when running the test it will be the dependency bundled with PHPUnit (deps installed by Composer or bundled in the PHAR) instead of your package one, which can weird to sly bugs. The use case of not being to use it on the other hand gives a flat out message and you're free to deal with this exceptional situation or not, as usually it's only temporary